submissions/Roophee/JS - DOM/index.js

```diff
@@ -0,0 +1,138 @@
+    },
+};
+
+
+// CREATE NAV
+
+let dynamicNav = function (e) {
+    e.preventDefault();
+    let fragment = document.createDocumentFragment();
+
+    for (let i of Object.keys(contentStorage)) {
+        let navItem = document.createElement("li");
+        let navLink = document.createElement("a");
+        let linkText = document.createElement("span");
+
+        navItem.className = "nav__item";
+        navLink.className = "nav__link";
+        navLink.href = "#top";
+        linkText.innerText = i;
+
+        navLink.appendChild(linkText);
+        navItem.appendChild(navLink);
+        fragment.appendChild(navItem);
+    }
+    navListOfElements.appendChild(fragment);
+}
+
+document.addEventListener('DOMContentLoaded', dynamicNav);
+
+// ADD CONTENT ON PAGE DOWNLOAD
+
+let addContentOnPageDownload = function (e) {
+    e.preventDefault();
+    content.innerHTML = defaultContent;
+}
+
+document.addEventListener("DOMContentLoaded", addContentOnPageDownload)
+
+// MOVE NAV & CONTENT
+
+function hideSideBar(e) {
+    e.preventDefault();
+    console.log(e);
+    deBar.classList.toggle('hide__sidebar');
+    burger.classList.toggle('burger__stop');
+};
+
+burger.addEventListener('click', hideSideBar);
+
+// ENTER-KEY EVENT SHOW-HIDE ".aside"
+
+burger.addEventListener('keydown', function (e) {
+    e.preventDefault();
+    if (e.keyCode == "13") {
+        deBar.classList.toggle('hide__sidebar');
+        burger.classList.toggle('burger__stop');
+    }
+}
+);
+
+// CLOSE ASIDE WHEN MOUSE CLICK OUT
+
+function hideSideBarClickAway(e) {
+    e.preventDefault();
+    if (!e.target.classList.contains('burger') && !deBar.contains(e.target)) {
+        if (deBar.classList.contains('hide__sidebar')) {
+            deBar.classList.toggle('hide__sidebar');
+            burger.classList.toggle('burger__stop');
+        };
+    };
+};
+
+document.addEventListener('click', hideSideBarClickAway);
+
+// ADD CONTENT IN ".content"
+
+function addContentOnClick(e) {
+
+    if (e.target && e.target.nodeName == 'A' || e.target.nodeName == 'SPAN') {
```

Comment:
Always use strict equality (===) for comparisons.

Reply:
Done.